security-force-monitor/sfm-cms
# Update install script with new dependency and command

## Overview

- Adds dependencies to install script, leftover from PR #812.
- Clarifies readme language for data archive instructions

## Testing Instructions
- checkout branch
- decrypt `scripts/after_install.sh`
- make sure the additions are as expected
